src/api/gitlab/gitlab.service.ts

```diff
@@ -10,8 +10,9 @@ import {
     IGitlabMergeRequestReaction,
     IGitlabMergeRequestDiscussion,
 } from './gitlab.interfaces';
-import Message from '../../helpers/Message';
+import { ISettingsModel } from '../mongo';
 /* eslint-enable no-unused-vars */
```

Comment:
Não quer tentar resolver esse problema na regra `no-unused-vars` para as interfaces?
Acho que [essa configuração](https://stackoverflow.com/questions/57802057/eslint-configuring-no-unused-vars-for-typescript) deve ajudar!